paper.md

```diff
@@ -115,7 +115,7 @@ packages used throughout the course are in Table 1 below.
 | | <p align="center">2</p> | <p align="center">Vectors, data frames, basic operations and use of functions</p>  | <p align="center">`tidyverse` [@tidyverse]</p> |
 | | <p align="center">3</p> | <p align="center">Exploratory analysis of datasets</p> | <p align="center">`tidyverse` [@tidyverse] |
 | | <p align="center">4</p> | <p align="center">Summary statistics and data visualization </p>| <p align="center">`tidyverse` |
-| | <p align="center">5</p> | <p align="center">Transforming datasets to different formats. Reproducing existing figures</p> | <p align="center">`tidyverse`</p> |
+| | <p align="center">5</p> | <p align="center">Transforming datasets to different formats, reproducing existing figures</p> | <p align="center">`tidyverse`</p> |
```

Comment:
Should this be "different formats, and reproducing existing"?

Reply:
I'm going to go through and make fuller revisions, so I can fix this up.